sig-scalability/slis/apimachinery_slis.md

```diff
@@ -0,0 +1,199 @@
+# API-machinery SLIs and SLOs
+
+The document was converted from [Google Doc]. Please refer to the original for
+extended commentary and discussion.
+
+***TODO(wojtek-t,deads2k,lavalamp): Get to consensus if we want to define latency
```

Comment:
I still think that for non-long-running-requests last byte matter more to clients/users than first byte.

@smarterclayton last byte is what our clients/controllers seem sensitive to today, right?